disable_odoo_online/models/publisher_warranty_contract.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2013 Therp BV (<http://therp.nl>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+
+class publisher_warranty_contract(models.AbstractModel):
+    _inherit = 'publisher_warranty.contract'
+
+    @api.multi
+    def update_notification(self, cron_mode=True, context=None):
```

Comment:
Asking myself whether this is legal for people using EE.
For them it is not allowed to temper with the notification system.
This makes this module IMO not usable for them while the other features could make sense. 
Would it be an option to split this function out of the module in a separate one?